### PR TITLE
US004 View Band Role in Job List

### PR DESCRIPTION
**Pull request by amchugh689**
- Created Band.java
- Created BandDao.java
- Created BandService.java
- Created BandController.java
- Created FailedToGetAllBandsException.java
- Created FailedToGetBandException.java
- Updated JobDao.java
- Implemented Unit Tests for Band - BandServiceTest.java

Screenshot 2023-12-14 at 09 25 56
Screenshot 2023-12-14 at 09 26 08
Screenshot 2023-12-14 at 09 26 34

### CODE REVIEW

**Review by Cormac-F — APPROVED** [review]
LGTM

**Comment by jkelso662** [comment]
Merge conflicts need sorted!

File: src/main/java/org/kainos/ea/DropwizardApplication.java [hunk 1]
```diff
@@ -30,6 +32,8 @@ protected SwaggerBundleConfiguration getSwaggerBundleConfiguration(DropwizardCon
     public void run(final DropwizardConfiguration configuration,
                     final Environment environment) {
         // TODO: implement application
```
Comment (jkelso662) [inline src/main/java/org/kainos/ea/DropwizardApplication.java]: You'll not need this comment as you have already implemented this below :)

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Great feature! Great code. Shame there are no unit tests anywhere 🙃

**Review by lullabelle — CHANGES_REQUESTED** [review]
checks are failing on this one. Also there is a merge conflict that will need resolved.
Are there any unit tests?

**Review by lullabelle — APPROVED** [review]

**Review by jkelso662 — APPROVED** [review]
Fab

**Pull request merged**